README.md

```diff
@@ -1,4 +1,6 @@
-# Cryptographic hashing functions for Dart
+## Cryptographic hashing functions for Dart
```

Comment:
It doesn't make sense to make this an h2 when it's the top-level header in the document.

Reply:
Removed header entirely. Department of redundancy department.